packages/ui/src/globals.css

```
@@ -1,2 +1,13 @@
+ * Tailwind v4, utilities only.
+ * Preflight/base is intentionally NOT emitted here: @gbl-uzh/ui is a library
+ * consumed inside apps that already ship preflight (via the design system),
+ * so shipping our own reset would double-apply. This preserves the previous
+ * Tailwind v3 `corePlugins.preflight: false` behaviour.
+ */
+@layer theme, base, components, utilities;
+
+@import 'tailwindcss/theme.css' layer(theme);
+@import 'tailwindcss/utilities.css' layer(utilities);
+
+@source './components/**/*.{ts,tsx}';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)